src/w3d/renderer/render2dsentence.cpp

```diff
@@ -781,7 +785,8 @@ void Render2DSentenceClass::Build_Sentence_Centered(const unichar_t *text, int *
                     ampersand_handled = true;
                 }
 
-                unichar_t cur_ch = *cur_text++;
+                unichar_t cur_ch = *cur_text;
+                ++cur_text;
```

Comment:
This is identical. Why change?

Reply:
It's easier on the eye in this case and compares with the decompile better.

Reply:
It will compile to exactly the same and is the normal convention for this sort of function